lib/Monad_WP/wp/WPI.thy

```diff
@@ -254,7 +254,7 @@ ML \<open>
     val frees = Term.add_frees t [];
     val frees' = Term.add_frees t' [];
   in
-   exists (member (=) frees') frees
+   exists (member (op =) frees') frees
```

Comment:
`op` is back?

Reply:
It never went away in ML, only in Isabelle. It's a miracle that this ever worked, but polyml allows it, it just complains that it is not legal ML but moves on. (A bunch of these got introduced by a too-global search and replace when `op` was removed in Isabelle)